docs/source/nlp/joint_intent_slot.rst

```diff
@@ -3,10 +3,253 @@
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| model.tokenizer.special_tokens            | string          | null                                                                             | Special tokens for the tokenizer (if they exist)                                                             |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| model.language_model.max_seq_length       | integer         | 50                                                                               | The maximum length of the input queries (in tokens)                                                          |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| model.language_model.pretrained_model_name| string          | bert-base-uncased                                                                | The pre-trained language model name (e.g. ``bert-base-cased`` or ``bert-base-uncased``)                      |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| model.language_model.lm_checkpoint        | string          | null                                                                             | The path to the pre-trained language-model checkpoint                                                        |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| model.language_model.config_file          | string          | null                                                                             | The path to the pre-trained language-model config file                                                       |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
```

Comment:
Please remove all tokenizer and language_model related params and add a link to the https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/nlp_model.rst#model-nlp

Reply:
Corrected